File: Tutorial_Sample_Code/Other/Multilingual/VB/VB_Win_Multilingual/Form1.vb

Imports CrystalDecisions.CrystalReports.Engine
Imports CrystalDecisions.Shared

Public Class Form1
    Private hierarchicalGroupingReport As Hierarchical_Grouping


    Private Sub ConfigureCrystalReports()
        hierarchicalGroupingReport = New Hierarchical_Grouping()
        myCrystalReportViewer.ReportSource = hierarchicalGroupingReport
        myCrystalReportViewer.ReportSource = hierarchicalGroupingReport
    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        ConfigureCrystalReports()
    End Sub
End Class

File: Tutorial_Sample_Code/Other/Multilingual/CS/CS_Win_Multilingual/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

using CrystalDecisions.CrystalReports.Engine;
using CrystalDecisions.Shared;


namespace CS_Win_Multilingual
{
	public partial class Form1 : Form
	{
		private Hierarchical_Grouping hierarchicalGroupingReport;

		
		public Form1()
		{
			InitializeComponent();
		}

		private void ConfigureCrystalReports()
		{
			hierarchicalGroupingReport = new Hierarchical_Grouping();
			crystalReportViewer.ReportSource = hierarchicalGroupingReport;
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			ConfigureCrystalReports();
		}

	}
}